src/main/java/jesseg/ibmi/opensource/nginx/NginxConfNode.java

```diff
@@ -161,7 +178,14 @@ void writeData(final PrintWriter _writer, final int _indent) {
         }
         try {
             for (final Entry<String, String> p : m_properties) {
-                _writer.println(childIndentStr + p.getKey() + " " + p.getValue() + ";");
+                String key = p.getKey();
+                String val = p.getValue();
+                if (val.isEmpty() || val == null) {
```

Comment:
With the Pattern/Matcher logic, this shouldn't be needed. The value will be an empty string and it will just kinda work out.

Reply:
What I was noticing before I added the separate case, was that the space between the key and value was still being written out so the semicolon would would have a space before it
```
location /basic_status {
      stub_status ;    
    }
```

instead of:

```
location /basic_status {
      stub_status;
    }

```

I simplified the logic here so that we build up the value string accordingly: 

```
for (final Entry<String, String> p : m_properties) {
    String valString = p.getValue().isEmpty() ? "" : " " + p.getValue();
    _writer.println(childIndentStr + p.getKey() + valString + ";");
}

```